.env.dev

```diff
@@ -1,5 +1,10 @@
 /* DEVELOPMENT */
 
+// Common
+
+REACT_APP_VERSION=$npm_package_version
```

Comment:
There are two ways to approach this. Like this or importing the package.json directly into the file. I think this is the cleaner way, which allows us to use version elsewhere in the app.